examples/pytorch/graphsage/main.py

```diff
@@ -133,48 +60,35 @@ def train_func(config: Dict):
             labels.append(label)
             losses.append(loss.item())
 
-        losses_full.extend(losses)
-        steps_in_epoch_trained = 0
-        if epoch % args.save_ckpt_by_epochs == 0:
-            save_checkpoint(model, logger, epoch, step, args)
+        if i % 10 == 0:
+            print(
```

Comment:
use logger for output.

Reply:
It needs to be print so it shows up in the log.txt in the test script.